packages/sources/dlc-btc-por/src/transport/utils.ts

```diff
@@ -126,3 +130,78 @@ export const getBitcoinNetwork = (networkName: string): BitcoinNetwork => {
 }
 
 export const FUNDED_STATUS = 1
+
+const bip32 = BIP32Factory(ellipticCurveCryptography)
+
+// Gets the derived public key from the extended public key.
+export const getDerivedPublicKey = (extendedPublicKey: string, bitcoinNetwork: Network): Buffer => {
```

Comment:
I think we still need to account for extended and compressed keys
1. for testnet (probably possible with just a length check here -- last I checked they hadn't updated testnet to extended)
2. on the off chance mainnet changes again

Reply:
It seems that they have updated the testnet as well and they use a new key format there as well, so now there is only one key format (extended)